# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/browser.js
# Commit: 6955906262c0
# Full Hash: 6955906262c0920b0f9f28dcf3177e9ce5e0dffc
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2020-04-07 03:53:09
# Regressor Bug: 1616353
# File Overlap Count: 1
# Description:
#   Bug 1616353 - Part 8: Remove forceInitialBrowserNonRemote, r=Gijs
#   
#   This method is no longer necessary, as frontend will always make the correct
#   decisions for the remoteness of the intial content browser by reading
#   openWindowInfo.
# ==============================================================================

diff -r a7700472807a -r 6955906262c0 browser/base/content/browser.js
--- a/browser/base/content/browser.js	Mon Apr 06 14:30:19 2020 +0000
+++ b/browser/base/content/browser.js	Mon Apr 06 14:30:22 2020 +0000
@@ -4994,13 +4994,6 @@
     ]);
   },
 
-  forceInitialBrowserNonRemote(aOpener) {
-    gBrowser.updateBrowserRemoteness(gBrowser.selectedBrowser, {
-      opener: aOpener,
-      remoteType: E10SUtils.NOT_REMOTE,
-    });
-  },
-
   setDefaultStatus(status) {
     this.defaultStatus = status;
     StatusPanel.update();